src/cli.ts

```diff
@@ -208,6 +243,13 @@ async function main(): Promise<void> {
                 'last 1 Safari versions',
             ],
         }
+
+        if (testsEnabled) {
+            packageJson.scripts = packageJson.scripts ?? {}
+            packageJson.scripts.test =
+                'TS_NODE_COMPILER_OPTIONS=\'{"module":"commonjs"}\' mocha --require ts-node/register --require source-map-support/register --recursive --watch-extensions ts, --timeout 200 \'src/**/*.test.ts\''
```

Comment:
I would suggest we configure Mocha in a package.json field, which I think is more readable and makes sure the settings are used even when mocha is launched directly from the terminal. https://mochajs.org/#configuration-format

Reply:
I made this change this, you're right: it's much more readable!